new-components/scanners/bandit/cmd/main.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/smithy-security/smithy/sdk/component"
 
-	"github.com/smithy-security/smithy/new-components/scanners/gosec/internal/transformer"
+	"github.com/smithy-security/smithy/new-components/scanners/bandit/internal/transformer"
```

Comment:
We were importing the wrong package here and failing to build as we can't import internal packages from other modules